### PR TITLE
main activity created

### PR DESCRIPTION
**Pull request by Shashank-1302**

### CODE REVIEW

**Comment by Shashank-1302** [comment]
Screenshot 2022-03-21 124602
Screenshot 2022-03-21 124627

**Review by MohitGupta121 — COMMENTED** [review]
Good work

File: app/src/main/res/drawable/ic_baseline_calendar_month_24.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#22367F"
+    android:viewportHeight="24" android:viewportWidth="24"
+    android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (MohitGupta121) [inline app/src/main/res/drawable/ic_baseline_calendar_month_24.xml:3]: Rename drawable with proper naming

File: app/src/main/res/drawable/ic_baseline_keyboard_arrow_down_24.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#22367F"
+    android:viewportHeight="24" android:viewportWidth="24"
```
Comment (MohitGupta121) [inline app/src/main/res/drawable/ic_baseline_keyboard_arrow_down_24.xml:2]: Rename this also like ic_arrow like that

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Comment by MohitGupta121** [comment]
@Shashank-1302 I fixed the layout file of mainactivity now check other errors according to Java-To-Kotlin Branch fix that and make build of that branch

**Comment by Shashank-1302** [comment]
Ok

**Review by MohitGupta121 — APPROVED** [review]

**Review by MohitGupta121 — APPROVED** [review]
Great Work, Thanks for your PR

**Pull request merged**